@planetarium/account/src/Account.ts

```diff
@@ -7,7 +7,7 @@ import Signature from "./Signature.js";
 export interface Account {
   getAddress(): Promise<Address>;
   getPublicKey(): Promise<PublicKey>;
-  sign(message: Message): Promise<Signature>;
+  sign(message: Message, isDigest?: boolean): Promise<Signature>;
```

Comment:
Isn't okay to update Account too?

Reply:
it's optional flag, so probably....

Reply:
After some discussions, we decided to just handle this inside AWS-KMS